apps/web/src/components/layout/components/v2/BridgeUpdateModal.tsx

```diff
@@ -33,14 +34,38 @@ export const BridgeUpdateModal: FC<BridgeUpdateModalProps> = ({ isOpen, toggleOp
     setUrl(event.target.value);
   };
 
+  const validateFromLocal = async (bridgeUrl: string): Promise<{ isValid: boolean }> => {
+    try {
+      const client = buildBridgeHTTPClient(bridgeUrl);
+      const response = await client.healthCheck();
+
+      const result = { isValid: response.status === 'ok' };
+
+      return result;
+    } catch {}
+
+    return { isValid: false };
+  };
+  const localDomains = ['localhost', '127.0.0.1'];
+  const isLocalAddress = () => {
```

Comment:
I've added this to check if the address is local, but this only works for conventional address. We cannot really target host file modifications. I'm not sure if this is really a clean solution.

Reply:
I think this can work for now